### PR TITLE
List: Add aria-label for allowItemDeleting: true and get rid of aria from nested buttons

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

**Review by alexanderPolosatov — DISMISSED** [review]

**Review by iBat — DISMISSED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js [hunk 1]
```diff
@@ -4289,3 +4257,91 @@ if(QUnit.urlParams['nojquery'] && QUnit.urlParams['shadowDom']) {
...
+            showSelectionControls: true
+        });
+
+        const $radioButtons = $(`.${LIST_SELECT_RADIOBUTTON_CLASS}`);
+
+        assert.strictEqual($radioButtons.attr('aria-label'), 'Check State');
+    });
+
+    [true, false].forEach(allowItemDeleting => {
+        QUnit.test(`List items element should have a correct aria-label when allowItemDeleting=${allowItemDeleting}`, function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js:4297]: Let's test allowItemDeleting  runtime change as well :)

File: packages/devextreme/js/ui/list/ui.list.edit.decorator.js [hunk 2]
```diff
@@ -82,6 +82,13 @@ const EditDecorator = Class.inherit({
         };
     },
 
+    _updateButtonAttributes($button) {
+        $button.attr({
+            role: null,
+            'aria-label': null,
+        });
```
Comment (ksercs) [inline packages/devextreme/js/ui/list/ui.list.edit.decorator.js:90]: cannot we set tabindex=-1 to exclude the element from the navigation sequence?

Comment (marker-dao) [reply packages/devextreme/js/ui/list/ui.list.edit.decorator.js:90]: You are right. It works.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js:4297]: [a4c09f24414ff140d8e8bf14a1b13b38b8d1935d]

File: packages/devextreme/js/ui/list/ui.list.edit.decorator.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/list/ui.list.edit.decorator.js:90]: [b6728ed285c5a9aeab22651a61475fa16f08cf09]

**Review by ksercs — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by iBat — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**